Quick typo fixes (#1189)
temporalio/documentation: docs/app-dev-context/handle-query.md

Queries are handled by your Workflow.

> Don’t include any logic that causes [Command](/concepts/what-is-a-command) generation within a Query handler (such as executing Activities). As this will lead to unexpected behavior.

Reviewer: ```suggestion
Don’t include any logic that causes [Command](/concepts/what-is-a-command) generation within a Query handler (such as executing Activities), as this will lead to unexpected behavior.
```

> Don’t include any logic that causes [Command](/concepts/what-is-a-command) generation within a Query handler (such as executing Activities), as this will lead to unexpected behavior.

Reviewer 2: ```suggestion
Don’t include any logic that causes [Command](/concepts/what-is-a-command) generation within a Query handler (such as executing Activities).
Including such logic causes unexpected behavior.
```
Or similar wording to avoid having the act of _not including_ leading to unexpected behavior.
